onelogin_rules/onelogin_active_login_activity.yml

```diff
@@ -0,0 +1,32 @@
+AnalysisType: rule
+Filename: onelogin_active_login_activity.py
+RuleID: OneLogin.ActiveLoginActivity
+DisplayName: OneLogin Active Login Activity
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Lateral Movement:Use Alternate Authentication Material
+Description: Multiple user accounts logged in from the same ip address.
+Threshold: 2
```

Comment:
Does this need to be done here or in the code itself? I have both right now, but I'm sure that is not what we want?

Reply:
Only here. You can also remove the manual DDB counting

Reply:
For a little bit of context @lindsey-w, support for alert thresholding in the rule metadata (i.e. by setting a `Threshold` meta data field) was only very recently added to Panther.

Before we had this feature, we had to use the dynamo key value store by calling `increment_counter` or `evaluate_threshold` and the like in any rule that needed thresholding. Now that we have built in support for thresholding, we can avoid calling out to dynamo for any simple detections that are just based on a counter and use the `Threshold` field instead. So in this case, just the `Threshold: 2` in the rule specification is sufficient, and any counting in the code itself can be removed.